static/js/ReaderTextTableOfContents.jsx

```diff
@@ -917,6 +917,9 @@ class ArrayMapNode extends Component {
     if ("refs" in this.props.schema && this.props.schema.refs.length) {
       var sectionLinks = this.props.schema.refs.map(function(ref, i) {
         i += this.props.schema.offset || 0;
+        if (ref == "") {
```

Comment:
```suggestion
        if (ref === "") {
```
Use === to make sure ref is actually `""` and not `null` or `false` or some other falsy value. See [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) for an explanation